### DISCUSSION TITLE
Guide for adding tools with Quarto

### DISCUSSION THREAD

**Discussion by RaymondBalise** [quarto-dev/quarto-cli#1674] [Q&A]
When I do `quarto install tool tinytex` I get a message saying:

```bash
(\) Updating Path (inspecting 1 possible paths)WARNING: Unable to determine a path to use when installing TeX Live. 
To complete the installation, please run the following:

/Users/me/Library/TinyTeX/bin/universal-darwin/tlmgr option sys_bin <bin_dir_on_path>
```

I don't know what to use for <bin_dir_on_path>.  I am guessing that it wants me to type `/Users/me/Library/TinyTeX/bin`.  Is that correct or does it want me to type the path to my machine's `/bin` directory?

Currently the quarto docs do not have a section on installing helper tools.  The PDF instructions have the line for installing tinytex and the instructions for chromium are buried on the page covering authoring diagrams. Having a page showing how to install/check the addins on Mac, Windows (and perhaps Linux), with the answer to the above question, would be most excellent.  

In particular, it would be good to have a gentle introduction (for frightened beginners who don't know what a command line is), dedicated to these lines:

```bash
quarto tools install tinytex
quarto tools install chromium
quarto tools check
```

**Comment by dragonstyle** [comment]
Hi -

Great feedback, thank you! I'll convert this to an issue to track improving the documentation as you describe.

As far as the specific warning that you're receiving, we need the `tlmgr` to be available globally in the terminal (somewhere on the system path). We try to figure out a directory on the bin path for you, but in this case we failed. You can see what directories are in your system path using the following command at the terminal:

`echo $PATH`

which should print a list of directories- use one of these as the `bin-dir-on-path`. A guess on on that might be present and work `/usr/bin`.

**Reply by RaymondBalise** [reply]
Cool.   I get

```bash
/usr/local/bin /usr/bin /bin /usr/sbin /sbin /opt/X11/bin /Library/Apple/usr/bin
```

So I will follow your advice and use `/usr/bin`.

Thank you so much for the fast feedback!

**Reply by dragonstyle** [reply]
I would use `/usr/local/bin` if I were you (and be sure that directory exists). I think later version of MacOS might protect `/usr/bin` ... Hope this helps... - ty for reporting will try to fix this so you don't need to mess with this!
